### ISSUE TITLE
Excel 2019: cannot report table row/column headers in protected sheets

### ISSUE THREAD

**Issue by Mister-Kayne** [nvaccess/nvda#18509]
In Excel 2019 on Windows 10 when a Work Sheet is protected, NVDA does not read the Column OR Row headings

**Comment by seanbudd** [comment]
Welcome @Mister-Kayne,

We are unable to process this issue as it stands.

Please open a new issue and fill out the bug issue template fully.

Kind Regards,
NV Access Software Developers

**Comment by Mister-Kayne** [comment]
Thank you for your comment and looking into this issue. I understand for some reason I did not fill out the complete bug template. I don’t recall it being presented to me to fill out. Having said that, perhaps being a new bee I must have gotten trigger happy and submitted an incomplete report. Sincere apologies, will raise a new one with all the details.

**Comment by seanbudd** [comment]
What steps are you following to open a new issue? The template appears when you click "Bug report" after "new issue"

Direct link: https://github.com/nvaccess/nvda/issues/new?template=bug_report.md

**Comment by Mister-Kayne** [comment]
I am going to re-attempt this. Unfortunately I cannot upload a video recording of the issue. So as suggested I will try my best in providing a detailed description on how to reproduce the issue and what is expected and how it is to work. Thank you for your guidance and patience.
